datanode/gateway/graphql/schema.graphql

```diff
@@ -1049,6 +1049,28 @@ type Query {
     "Optional party IDs to filter for"
     partyIDs: [String!]
   ): PaidLiquidityFeesConnection
+
+  "Get total transfer fee discount available"
+  totalTransferFeeDiscount(
+    "ID of party eligible for the discount."
+    partyId: String!
+    "ID of asset to associated with the discount."
+    assetId: String!
+  ): TotalTransferFeeDiscount
+
+  "Estimate transaction fee"
```

Comment:
Would this be more consistent with other parts of the docs?

"Get transaction fee estimate"